executor/common.h

```diff
@@ -158,12 +164,10 @@ static uint64 current_time_ms(void)
 	return (uint64)ts.tv_sec * 1000 + (uint64)ts.tv_nsec / 1000000;
 }
 #endif
-
```

Comment:
Please refrain from unrelated edits.

Reply:
I'm extremely sorry and will make sure that I double check the next time I send a request/ PR.